test/unit/host_test.rb

```diff
@@ -710,8 +710,6 @@ def teardown
       hostgroup = FactoryGirl.create(:hostgroup, :with_puppet_orchestration)
       h = FactoryGirl.create(:host, :managed, :with_environment, :hostgroup => hostgroup)
       Setting[:manage_puppetca] = true
-      assert h.puppet_proxy.present?
-      assert h.puppetca?
```

Comment:
This patch hasn't been merged, how could it have broken anything yet?  Do you mean 5e1f3b7?

This patch is related, but does quite a different thing.